### ISSUE TITLE
Update to CLDR v29

### ISSUE THREAD

**Issue by bojanz** [punic/punic#81]
Punic ships with v27.

**Comment by mlocati** [comment]
Last time I checked (v28) there were no changes.
Do you know if there's something new/changed in v29?

**Comment by bojanz** [comment]
I've never had a no-change update.

https://github.com/commerceguys/intl/commit/5bfa07e0aad1d295dd0a85d0091d382175110348
https://github.com/commerceguys/intl/commit/b17d58620ec0cb0c51e11980c8f0b4f5860b8fbf

**Comment by mlocati** [comment]
I tested the latest version, and it seems that the latest `ldml2json` tool is quite crappy...

For instance, here's the content of the generated `measurementData.json` file:

```json
{
  "supplemental": {
    "version": {
      "_number": "$Revision: 12447 $",
      "_unicodeVersion": "8.0.0",
      "_cldrVersion": "29"
    },
    "measurementData": {
      "measurementSystem": {
        "001": "metric"
      },
      "temperature": {
        "LR": "metric",
        "MM": "metric"
      },
      "measurementSystem": {
        "GB": "UK",
        "LR": "US",
        "MM": "US",
        "US": "US"
      },
      "temperature": {
        "BS": "US",
        "BZ": "US",
        "KY": "US",
        "PR": "US",
        "PW": "US"
      },
      "paperSize": {
        "001": "A4",
        "BZ": "US-Letter",
        "CA": "US-Letter",
        "CL": "US-Letter",
        "CO": "US-Letter",
        "CR": "US-Letter",
        "GT": "US-Letter",
        "MX": "US-Letter",
        "NI": "US-Letter",
        "PA": "US-Letter",
        "PH": "US-Letter",
        "PR": "US-Letter",
        "SV": "US-Letter",
        "US": "US-Letter",
        "VE": "US-Letter"
      }
    }
  }
}
```

As you can see, the generated JSON is malformed (duplicated keys).

I think that I'll need to generate the Punic JSON data starting from the source XML files directly, but it'll require some time.

**Comment by bojanz** [comment]
Oh, wow. That's worth contacting CLDR, their git repo has malformed data.

**Comment by mlocati** [comment]
Done: https://github.com/unicode-cldr/cldr-json/issues/11

**Comment by mlocati** [comment]
http://unicode.org/cldr/trac/ticket/9405

**Comment by mlocati** [comment]
http://unicode.org/cldr/trac/ticket/9405 has been resolved for CLDR 30.0.3.

BTW, I met two other issues when trying to update Punic data:

- http://unicode.org/cldr/trac/ticket/10044 (minor)
- http://unicode.org/cldr/trac/ticket/10045 (blocking)

**Comment by mlocati** [comment]
We upgraded to CLDR 31 in Punic version 2.0
